bundles/org.openhab.core.compat1x/src/main/java/org/openhab/model/sitemap/Chart.java

```diff
@@ -27,6 +27,7 @@
  * @see org.openhab.model.sitemap.SitemapPackage#getChart()
  * @model
  * @generated
+ * @author Kai Kreuzer - Initial contribution
```

Comment:
Usually, src/gen/java is for dynamically generated source files - these here were generated once and then copied around and committed. Maybe the easiest solution is to simply keep the author tags, considering that almost all class files have once been generated out of our archetype ;-)